src/test/java/io/strimzi/test/container/StrimziKafkaKraftContainerIT.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test.container;
+
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class StrimziKafkaKraftContainerIT {
+
+    private static final Logger LOGGER = LogManager.getLogger(StrimziKafkaKraftContainerIT.class);
+
+    private StrimziKafkaContainer systemUnderTest;
+
+    private void assumeDocker() {
+        Assumptions.assumeTrue(System.getenv("DOCKER_CMD") == null || "docker".equals(System.getenv("DOCKER_CMD")));
+    }
+
+    @Test
+    void testStartContainerWithEmptyConfiguration() throws ExecutionException, InterruptedException {
+        assumeDocker();
+        systemUnderTest = StrimziKafkaContainer.createWithKraft(1)
+            .waitingFor(Wait.forLogMessage(".*Transitioning from RECOVERY to RUNNING.*", 1));
```

Comment:
Shouldn't we encapsulate this via some `waitForRunning` method? It would save users having to know the regex, and we could insulate them should the log message ever change.